testsuite/common.h

```diff
@@ -2,20 +2,11 @@
 #include <storage/StorageInterface.h>
 
 
-namespace storage
-{
-
-    struct TestEnvironment : public Environment
-    {
-	TestEnvironment() : Environment(false)
-	{
-	    testmode = true;
-	    autodetect = false;
-	    logdir = testdir = "tmp";
-	}
-    };
+#define TRACE() cout << "### " << __FUNCTION__ << "()" << endl
```

Comment:
Name is not much descriptive. I expect that it activate printing called methods.
Maybe something like `PRINT_METHOD` ?